packages/frontend/src/components/common/balance/FiatBalance.tsx

```diff
@@ -11,7 +11,7 @@ type FiatBalanceProps = {
     showSignUSD?: boolean,
     nearTokenFiatValueUSD?: number,
     isNear?: boolean,
-    decimals: number,
+    decimals?: number,
```

Comment:
Is there a reason we make this becomes optional? Decimals is a standard property in FT standard (https://github.com/near/NEPs/blob/master/neps/nep-0148.md)

Reply:
For this component it's not always being put through as a prop